languoids/tree/indo1319/indo1320/indo1321/indo1323/oriy1254/gaud1237/gaud1238/uncl1515/kurm1243/md.ini

```diff
@@ -5,7 +5,7 @@ hid = kfv
 level = language
 iso639-3 = kfv
 latitude = 26.9175
-longitude = 87.3205
+longitude = 87.32050000000001
```

Comment:
This one - and the `latitude = 1.9144999999999999` above - keep coming back. Are coordinates refreshed automatically on your end?